### PR TITLE
Revise the associated phrases feature

### PR DESCRIPTION
**Pull request by zonble**
- Now the shift + enter key to insert associated phrases is on by default.
- When the "associated phrase" mode menu item is selected, the associated phrases would appear when each character is entered.
- Also added tests.

### CODE REVIEW

File: McBopomofoTests/AssociatedPhrasesTests.swift [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import XCTest
+
+@testable import McBopomofo
+
+class AssociatedPhrasesTests: XCTestCase {
+
+    var handler = KeyHandler()
+
+    override func setUpWithError() throws {
+        Preferences.chineseConversionEnabled = false
```
Comment (lukhnos) [inline McBopomofoTests/AssociatedPhrasesTests.swift:33]: Question / nit: if a developer runs the test locally, would this disable their Chinese conversion without their knowing?

Comment (lukhnos) [inline McBopomofoTests/AssociatedPhrasesTests.swift:35]: Question / nit: if a developer runs the test locally, would this disable their Chinese conversion without their knowing?

**Review by lukhnos — APPROVED** [review]
LGTM! Just one question/nit about the test setup.

File: McBopomofoTests/AssociatedPhrasesTests.swift (hunk 1 above)
Comment (zonble) [reply McBopomofoTests/AssociatedPhrasesTests.swift:35]: Okay. I Will fix this later.

**Pull request merged**